packages/waku/src/lib/utils/vite-loader.ts

```diff
@@ -1,12 +1,31 @@
 import { createServer as createViteServer } from 'vite';
 import { fileURLToFilePath } from '../utils/path.js';
+import { typegenPlugin } from '../plugins/vite-plugin-typegen.js';
+import type { Config } from 'waku/config';
 
 export const loadServerFile = async (fileURL: string) => {
+  // temp server to load the  waku config to see if typegen is enabled
+  const viteTempServer = await createViteServer({
+    ssr: {
+      external: ['waku'],
+    },
+  });
+
+  let config: Config = {};
+  try {
+    config = (await viteTempServer.ssrLoadModule(fileURLToFilePath(fileURL)))
+      .default;
+  } finally {
+    await viteTempServer.close();
+  }
```

Comment:
I don't particularly like needing to do this temp server thing, but I needed a way to load `config.typegen` before adding the plugin. Is there a better way to do this?

Reply:
Does removing the config solve it?

Reply:
yes, thanks!